src/test/java/wellnus/atomichabit/AtomicHabitTest.java

```diff
@@ -89,6 +89,21 @@ public void addHabit_invalidCommand_atomicHabitExceptionThrown() {
                 + command);
     }
 
+    /**
+     * Test AddCommand to throw {@link AtomicHabitException} when a duplicate habit is added.
+     *
+     * @throws WellNusException
+     */
+    @Test
+    public void addHabit_duplicateHabit_atomicHabitExceptionThrown() throws WellNusException {
+        String payload = "junit test";
+        String testAddCommand = String.format("%s --name %s", ADD_HABIT_COMMAND, payload + System.lineSeparator());
+        HashMap<String, String> arguments = parser.parseUserInput(testAddCommand);
+        Command addCommand = new AddCommand(arguments, habitList);
+        addCommand.execute();
```

Comment:
Hmm i think this is unnecessary cos the next line(`asertThrows`) already calls this method for you. Then you can avoid adding the `throws WellnusException` clause for this method

Reply:
Require this part to simulate adding 2 duplicate habits.

Reply:
My bad, misunderstood the intent here. resolved!